### PR TITLE
fix: add Divyansh Raj

### PR DESCRIPTION
**Pull request by divyanshraj0408**
## Changes proposed
Added my profile

## Check List (Check all the applicable boxes) 

- [X] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [X] All new and existing tests passed.
- [X] This PR does not contain plagiarized content.
- [X] The title of my pull request is a short description of the requested changes.

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/903"></a>

### CODE REVIEW

**Review by syedsohan — CHANGES_REQUESTED** [review]
Kindly revert the `package-lock.json` file.

**Review by syedsohan — CHANGES_REQUESTED** [review]
Change the PR title to - `fix: add Divyansh Raj`

File: public/data/divyanshraj0408.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
+{
+  "name": "Divyansh Raj",
+  "type": "personal",
+  "bio": "Open Source Enthusiast!||Pursuing highschool diploma||Doing 100DaysOfCode.",
```
Comment (syedsohan) [inline public/data/divyanshraj0408.json:4]:
```suggestion
  "bio": "Open Source Enthusiast! | Pursuing high school diploma | Doing 100DaysOfCode.",
```

File: public/data/divyanshraj0408.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/divyanshraj04",
+      "icon": "twitter"
+    }
+  ],
+  "milestones": [
+    {
+      "title": "Participated in and completed HactoberFest2021",
```
Comment (syedsohan) [inline public/data/divyanshraj0408.json:20]:
```suggestion
      "title": "Participated and completed Hactoberfest 2021.",
```

File: public/data/divyanshraj0408.json [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+      "date": "October 2021",
+      "icon": "Globe",
```
Comment (syedsohan) [inline public/data/divyanshraj0408.json:22]:
```suggestion
      "icon": "globe",
```

**Review by syedsohan — CHANGES_REQUESTED** [review]

**Comment by divyanshraj0408** [comment]
> Kindly revert the `package-lock.json` file.

Thanks for all the great and helpful suggestions. How can I revert the package-lock.json file, do I have to delete it?

**Comment by syedsohan** [comment]
> > Kindly revert the `package-lock.json` file.
> 
> Thanks for all the great and helpful suggestions. How can I revert the package-lock.json file, do I have to delete it?

- Copy the current `package-lock.json` from the `main branch` into `your branch` and then `replace` and `commit` the changes.

**Review by Panquesito7 — APPROVED** [review]
LGTM 🚀 Thank you for your contribution! 😄👍

**Review by syedsohan — APPROVED** [review]

**Review by syedsohan — APPROVED** [review]

**Pull request merged**